src/BenchmarkDotNet/Environments/HostEnvironmentInfo.cs

```diff
@@ -96,7 +96,7 @@ public override IEnumerable<string> ToFormattedString()
                 yield return $"Frequency={ChronometerFrequency}, Resolution={ChronometerResolution.ToString(cultureInfo)}, Timer={HardwareTimerKind.ToString().ToUpper()}";
 
             if (RuntimeInformation.IsNetCore && IsDotNetCliInstalled())
-                yield return $".NET Core SDK={DotNetSdkVersion.Value}";
+                yield return $".NET SDK={DotNetSdkVersion.Value}";
```

Comment:
We should display `.NET SDK` only for 5.0+; otherwise we should display `.NET Core SDK`.

Reply:
We can't just use `if (Version.TryParse(DotNetSdkVersion.Value, out var sdkVersion) && sdkVersion >= new Version(5,0))` because the SDK version contains words like "preview" in the name and 5 segements (`A.B.C.D.E`). Example: `5.0.100-preview.8.20362.3`. Anyway I have added a check for that